### PR TITLE
Use newest version of clerk-druid-extensions.

### PR DESCRIPTION
**Pull request by soenke-tippelmann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Samza, Kafka, Storm, Spark, and Hadoop.
 1. Rebase this branch (mbr-master) on the latest druid version tag you wanna use (e.g. git rebase --onto druid-0.9.1.1 druid/master).
 2. Add `mbr` suffix to version number (e.g. 0.9.1.1-mbr). You have to update the version number in all build files.
 3. Build project with `mvn clean package -DskipTests` (tests would take ages).
-4. Copy the artifact (`distribution/target/druid-0.9.1.1-mbr2-bin.tar.gz`) to `mastodon-master-1.hadoop.ams2.m6r.eu:/var/www/druid/`.
+4. Copy the artifact (`distribution/target/druid-0.9.1.1-mbr13-bin.tar.gz`) to `mastodon-master-1.hadoop.ams2.m6r.eu:/var/www/druid/`.
```
Comment (jkukul) [inline README.md]: Maybe it's better to make this instruction more generic instead, so that it doesn't have to be modified on every version change?

Comment (soenke-tippelmann) [inline README.md]: Well, it is a simple replace-all using sed or sth similar, so there is no extra work involved. I'm thinking about providing a script that does this version increase automatically, but haven't done so yet. Wdyt?

**Comment by soenke-tippelmann** [comment]
Its mbr13, since I have to increment the number every time I update the extensions and attempt a rebuild. I just made one commit instead of 13, because they would not contain any additional information...

**Comment by gesundkrank** [comment]
As I just told you: Please use snapshot versions for testing. E.g. the current test version should be mbr3-SNAPSHOT. They can be overwritten. 

And please remove your test versions from our release repo (It's possible via the web interface).

**Comment by jkukul** [comment]
@soenke-tippelmann Remember to delete the branch :)

**Pull request merged**